idl/github.com/uber/cadence/shared.thrift

```diff
@@ -254,6 +254,15 @@ enum ArchivalStatus {
   ENABLED,
 }
 
+enum IndexedValueType {
+  STRING,
+  KEYWORD,
```

Comment:
what is the difference between STRING and KEYWORD?

Reply:
Keyword is can only be used as exact match when query, 
for example UUID "1234-567-XXX-XXX" will be matched when query "567" when it saved as STRING Type, but not be matched when saved as KEYWORD.